### ISSUE TITLE
Integrate a lora module

### ISSUE THREAD

**Issue by htotoo** [htotoo/ESP32-Portapack#4]
Add lora module, to get Meshtastic (kinda) work on it too.

**Comment by ucflumm** [comment]
Maybe to track ExpressLRS drone signals?

**Comment by htotoo** [comment]
Good idea, but maybe seems too specific for some regions, and not for "all".

**Comment by htotoo** [comment]
Nowadays i'm thinking about adding Meshtastic (rx, unencrypted) support as the lora addition. But it will be one of the last additions! (if i don't change my mind during that time..)

**Comment by fluffrider** [comment]
Do it! Setting a Meshtastic network here in the rural desert!On Dec 28, 2024, at 12:02 PM, Totoo ***@***.***> wrote:﻿
Nowadays i'm thinking about adding Meshtastic (rx, unencrypted) support as the lora addition. But it will be one of the last additions! (if i don't change my mind during that time..)
